lib/tasks/model.rake

```diff
@@ -1,83 +0,0 @@
-# TRANSLATORS: do not translate
-desc <<~EOT
-  When Foreman imports a new host by reading its facts it will create the host and
-  set its model to a value based upon the facts. This can result in a multiplicity
-  of similarly named models. These may be condensed donw into a few basic models
```

Comment:
Did you find this when looking for typos? ;)

Reply:
Actually when looking for places that still check for `SETTINGS[:unattended]`